docs/sources/reference/commandline/cli.md

```diff
@@ -1136,6 +1138,20 @@ uses up the `VIRTUAL SIZE` listed only once.
     tryout                        latest              2629d1fa0b81b222fca63371ca16cbf6a0772d07759ff80e8d1369b926940074   23 hours ago        131.5 MB
     <none>                        <none>              5ed6274db6ceb2397844896966ea239290555e74ef307030ebb01ff91b1914df   24 hours ago        1.089 GB
 
+#### Listing image digests
+
```

Comment:
Images that use the v2 or later format, have a content-addressable identifiers called a *digest*.  As long as the input used to generate the image is unchanged, the digest value is predictable.  To list image digest values, use the `--digests` flag: 

<move command a listing here>

When pushing or pulling to a 2.0 registry, the `push`or `pull` command output concludes with the image digest. 
 You can `pull` using a digest value.  You can also reference by digest in `create`,`run`, and `rmi` commands.